rdt/transformers/numerical.py

```diff
@@ -198,6 +198,38 @@ def _reverse_transform(self, data):
 
         return data.astype(self._dtype)
 
+    def _set_fitted_parameters(
+        self,
+        column_name,
+        null_transformer,
+        rounding_digits=None,
+        min_max_values=None,
+        dtype='object',
+    ):
+        """Manually set the parameters on the transformer to get it into a fitted state.
+        Args:
+            column_name (str):
+                The name of the column to use for the transformer.
+            null_transformer (NullTransformer):
+                A fitted null transformer instance that can be used to generate
+                null values for the column.
+            min_max_values (Tuple(float) or None):
+                None or a tuple containing the (min, max) values for the transformer.
+            rounding_digits (int or None):
+                The number of digits to round to.
+            dtype (str):
+                The pandas dtype the reversed data should be converted to
+        """
+        self.null_transformer = null_transformer
+        self.columns = [column_name]
+        self.output_columns = [column_name]
+        if min_max_values:
+            self._min_value = min(min_max_values)
+            self._max_value = max(min_max_values)
```

Comment:
If `self.enforce_min_max_values` is `True` we should error if these aren't provided